### PR TITLE
Update IAM mastery badge lvl2 competencies

### PR DESCRIPTION
**Pull request by hndoss**

### CODE REVIEW

File: ops/aws/IAM-level2.md [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
-- Understand IAM concepts of:
-   - Policies
-   - Users
-   - Groups
-   - Access Keys
-   - Roles
-- Be able to create an IAM user and setup the CLI
-- Be able to create a user and assign roles, policies and groups
+
+* Be able to create roles with trusted entities
```
Comment (bsommardahl) [inline ops/aws/IAM-level2.md:5]: Remove `Be able to`

File: ops/aws/IAM-level2.md [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
+* Write policies with granular permissions
+* Be able to create an instance profile for EC2 instances
```
Comment (bsommardahl) [inline ops/aws/IAM-level2.md:7]: Remove `Be able to`

File: ops/aws/IAM-level2.md (hunk 1 above)
Comment (hndoss) [reply ops/aws/IAM-level2.md]: Removed

File: ops/aws/IAM-level2.md (hunk 2 above)
Comment (hndoss) [reply ops/aws/IAM-level2.md]: Removed

**Pull request merged**